main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -264,7 +266,7 @@ object Packager {
       addToZip(zip, "META-INF/MANIFEST.MF", manifest.getBytes)
 
       // Add all class files.
-      for (buildFile <- getAllFiles(getBuildDirectory(p))) {
+      for (buildFile <- getAllFiles(getBuildDirectory(p)).sorted(PathComparator)) {
```

Comment:
This sorts based on the path of the class file. But would it not be enough to sort based on `fileName` further down?

The idea being that if you have:

```
/home/magnus/flix/mypackage/foo.flix
```

and

```
/home/kengo/flix/mypackage/foo.flix
```

then presumably only the `mypackage/foo.flix` part matters since this is the name that goes in the zip file?

Reply:
In other words, I would compute "fileNameWithSlashes" first and then sort based on those. 

This should probably be a list of pairs or something like this.